stable/traefik/templates/deployment.yaml

```diff
@@ -59,6 +59,10 @@ spec:
           periodSeconds: 10
           successThreshold: 1
           timeoutSeconds: 2
+        {{- if .Values.env }}
+        env:
+{{ toYaml .Values.env | indent 8 }}
```

Comment:
@krancour ok I refreshed my memory on this (it's been a while :)). My original intent was to have the sensitive environment variables be drawn from a secret. e.g.:

```
  - name: AWS_REGION
    value: us-east-1
  - name: AWS_ACCESS_KEY_ID
    valueFrom:
        secretKeyRef:
            name: aws
            key: ACCESS_KEY_ID
  - name: AWS_SECRET_ACCESS_KEY
    valueFrom:
        secretKeyRef:
            name: aws
            key: SECRET_ACCESS_KEY
```

Maybe I can document the intended usage in `templates/NOTES.txt`?